hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -72,27 +72,25 @@ class BaseMIL(LightningContainer, EncoderParams, PoolingParams, LossCallbackPara
                                                              "generating outputs.")
     maximise_primary_metric: bool = param.Boolean(True, doc="Whether the primary validation metric should be "
                                                             "maximised (otherwise minimised).")
-    tune_classifier: bool = param.Boolean(
```

Comment:
For consistency with the rest of the network, encapsulate classifier specific parameters in ClassifierParams class